src/widgets/vitals/vitals-header/vital-header-details.compont.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
this file has a spelling error, please rename vital-header-details.component.tsx